contracts/seed/Seed.sol

```diff
@@ -394,6 +394,26 @@ contract Seed {
         emit TokensClaimed(_funder, _claimAmount);
     }
 
+    function claimTip() external returns (uint256) {
+        require(
+            endTime < block.timestamp || maximumReached,
+            "Seed: the distribution has not yet finished"
+        );
+
+        uint256 amountClaimable;
+
+        amountClaimable = calculateClaimBeneficiary();
+        require(amountClaimable > 0, "Seed: amount claimable is 0");
+
+        tip.totalClaimed += amountClaimable;
```

Comment:
```suggestion
        tip.totalClaimed = amountClaimable;
```

One can only claimTip once, no?
The `+=` suggests, that this function will be called multiple times.
Can just `=` work?

Reply:
+= is correct, because of vesting setup